package.json

```diff
@@ -46,12 +47,12 @@
     "@babel/core": "^7.25.2",
     "@babel/eslint-parser": "^7.25.1",
     "@babel/generator": "^7.25.0",
+    "@babel/plugin-syntax-typescript": "^7.25.4",
     "@babel/plugin-transform-regenerator": "^7.24.7",
     "@babel/preset-env": "^7.25.3",
     "@babel/preset-flow": "^7.24.7",
     "@definitelytyped/dtslint": "^0.0.127",
-    "@jest/create-cache-key-function": "^29.6.3",
-    "@nx/js": "^21.2.4",
+    "@jest/create-cache-key-function": "^29.7.0",
```

Comment:
We don't want to remove `@nx/js` from here

Reply:
Added back